submissions/YuraZagor/hooliPopup/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>myPopup</title>
+</head>
+<body>
+	<nav>
+		<ul class="ul--menu">
+			<li class="li--menu"><a href="#">Mail</a></li>
+			<li class="li--menu"><a href="#">Images</a></li>
+
+			<li class="li--menu popup pic" >
+				<label id="popup-label" class="sub-menu" for='checkbox-icon'> <img src="./popup-icons/popup-button.png" alt="popup-label" > </label>
+				<input type="checkbox" id="checkbox-icon" name='checkbox-icon' class='checkbox-icon' alt="popup-checkbox" >
```

Comment:
Add styles on the focus state, to notify the user where the page focus is. [Watch video](https://share.vidyard.com/watch/cJKtB6pUbCL6JqXeP6gZif?)

Reply:
I really tried a bunch of aproaches to outline the menu.  Finally I subjected both labels to  the  .show--all--label  ruleset  -- as it works for _.show--all_ input  -- and it wouldn't work for popup.  I tried by tag, by class and by id,   +  !important;  and it digitdn't help neither.  My logic breaks here how can one label work while the other doesn't

Reply:
Change the order, the label should be after the input, and then using CSS you can add styles for the label.

Reply:
Thanks, now it's working.  I plan to read more on that matter